### ISSUE TITLE
Search with word "new" crashes

### ISSUE THREAD

**Issue by CVettai** [rochus-keller/CrossLine#11]
I am using Windows 10, latest version of Crossline
Summary: Query crashes when there is word "new" (with out double quotes)
The language setting of Windows is English(US)

Steps to reproduce
It is simple.
1. Bring in the find window
2. Type new in the query field 
3. Press Enter
4. Crossline vanishes

**Comment by rochus-keller** [comment]
Thanks for the hint. I will check this.

**Comment by rochus-keller** [comment]
Meanwhile I made tests on Windows 10, both x86 and x64, but I'm not able to reproduce the crash.

I downloaded the crossline win32 x86 and x64 binaries using the links in the readme and also the crossline demo database.

Then I opened the x86 version with the demo and searched for "new" (without quotes); it builds the index as expected and then shows the results. No crash.

Then I did the same thing with the x64 version. Same result.

I also tried on Linux x86, neither a crash.

Can you please try with the demo database and tell me, whether you still see a crash? If no, are you able to create a minimum version of your crossline database where you can reproduce the crash?

**Comment by CVettai** [comment]
You are right. I am also not able to reproduce it with a fresh db. So, it seems something data specific to my existing db. I updated the indices, but still it is reproducible. Can some logs help here to know what is happening? The query term "new" is not super critical for me. But, I moved all my data to this fantastic program a while back, just concerned  if it may start behaving similarly on more query terms. Thank you

**Comment by rochus-keller** [comment]
Did you notice that you can export outlines to e.g. HTML? You could try to search in the browser to find out where the "new" are located; it might be possible that the crash happens when displaying the search result in CrossLine; or you could restrict your search to the present outline; maybe you make a copy of an outline where the crash occurs and remove lines until only the line is left which crashes; then you could remove private information and export this line to cldx format (context menu "export outline") and send me the cldx, so I could reproduce the crash in the debugger.

**Comment by CVettai** [comment]
Sorry for the delay. The problem started appearing for many keywords to the point the software started become unusable. I deleted the whole index, and then rebuilt it. This seems to solve the problem for now. I will update here on how it goes.

**Comment by rochus-keller** [comment]
That's interesting. I never had an issue with the index so far. Is your hard drive OK? Do you use an SSD or a magnetic drive?

**Comment by CVettai** [comment]
Hard drive is good as far as I know - no data corruption or such issues I am aware of. It is a magnetic drive. I should admit there has been system shutdowns while crossline was still running.

**Comment by rochus-keller** [comment]
I will check under what circumstances the index file can get out of sync and what effect it has. You can delete index files at any time and Crossline will automatically rebuild these if not present.
